opencti-platform/opencti-front/src/private/components/common/form/AuthorizedMembersField.tsx

```diff
@@ -318,7 +318,7 @@ const AuthorizedMembersField = ({
                   </Field>
                   <IconButton
                     color="secondary"
-                    aria-tag="More"
+                    aria-label="More"
```

Comment:
is this change to avoid a console error?

Reply:
yes ! `aria-tag` does not exist actually.